server/src/main/java/io/druid/server/coordination/DruidServerMetadata.java

```diff
@@ -27,7 +27,8 @@
 public class DruidServerMetadata
 {
   private final String name;
-  private final String host;
+  private final String hostAndPort;
+  private final String hostAndTlsPort;
```

Comment:
Do we need to keep these concatenated strings? Maybe better to keep host, port, and tlsPort individually?